packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -106,6 +106,7 @@ export class ReadAlongComponent {
   current_page;
   hasTextTranslations: boolean = false;
   @State() images: object;
+  @State() translations: object;
```

Comment:
Could use an index signature here (... and for `images` too while we're at it)

Reply:
good point too. thanks @dhdaines - I'll make these changes.